File: Dreya/messenger/Messenger.java
package dreya.messenger;


/**
 *
 * @author smv
 */
public class Messenger {
    
    public Messenger()
    {
    }
    

    public static void sendMessage(String recipient,String message)
    {
        Sender sender = new Sender();
        sender.send(recipient, message);
        //System.out.println(message);
    }
        
}

File: Dreya/Messenger/Messenger.py
# -*- coding: utf-8 -*-

from Http import Http
import urllib

class Messenger:


    def sendMessage(recipient = "", content = ""):

        if content:
            import json
            from Config import Config
            json = urllib.parse.quote(str(json.dumps({"recipient": recipient, "message": content})))
            payload = {"payload": json}
            Http.post(Config.messengerSenderGateway, payload)
